public/views/swagger.html

```diff
@@ -1,30 +1,34 @@
 <!-- HTML for static distribution bundle build -->
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
```

Comment:
most of the changes are from running `format` -- but I think we want that, so i kept the changes.